dataset/src/test/scala/frameless/ColumnTests.scala

```diff
@@ -14,9 +14,8 @@ final class ColumnTests extends TypedDatasetSuite with Matchers {
   private implicit object OrderingImplicits {
     implicit val sqlDateOrdering: Ordering[SQLDate] = Ordering.by(_.days)
     implicit val sqlTimestmapOrdering: Ordering[SQLTimestamp] = Ordering.by(_.us)
-    implicit val arbInstant: Arbitrary[Instant] = Arbitrary(
-      Gen.chooseNum(0L, Instant.MAX.getEpochSecond)
-        .map(Instant.ofEpochSecond))
+    implicit val arbInstant: Arbitrary[Instant] =
+      Arbitrary(Gen.choose[Instant](Instant.EPOCH, Instant.now()))
```

Comment:
Wondering how that was handled before?

Reply:
https://github.com/typelevel/frameless/pull/581/files/2d05b62a70067b890eab5813ad4b17565481d52f#r747134559